resolwe_bio/processes/jbrowse/variants.yml

```diff
@@ -0,0 +1,39 @@
+# =========================
+# JBrowse tracks - Variants
+# =========================
+#
+---
+
+- slug: jbrowse-vcf-track
```

Comment:
I understand your concerns, but a general solution could visualize all variant files on the server, whereas the optimized solution requires a special pipeline. @jkokosar Do you see an advantage in generality?